bootstrap.sh

```diff
@@ -9,21 +9,30 @@ if [ "$1" = "--skip_root_installs" ]; then
   SKIP_ROOT_INSTALLS=True
 fi
 
+# Exits if $1 is not 0 and prints $2 as error message.
+function fail_on_error() {
+  local rc=$1
+  local msg=$2
+  if [ -z "$msg" ]; then
+    rc=1
+    msg="missing arguments in fail_on_error() call in bootstrap.sh."
+  fi
+  if [ $rc -ne 0 ]; then
+    echo "ERROR: $msg"
+    exit $rc
+  fi
+}
+
 if [ ! -f bootstrap.sh ]; then
```

Comment:
You're right. Not sure what I did wrong before when testing it with single brace, but it works.